bunyan-pretty.config.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const { format } = require('util');
+const { prettify } = require('sql-log-prettifier');
+
+module.exports = {
+  level: 'debug',
+  customPrettifiers: {
+    query(input) {
+      const index = input.indexOf(': ') + 1;
+      const sql = input.substring(index);
+      return format('\n```sql\n%s\n```', prettify(sql));
```

Comment:
`prettify` accepts `settings` as the second argument, have you considered changing some colors?

Reply:
This is POC in its true meaning so likely some things should be changed 😀